app/controllers/v0/education_benefits_claims_controller.rb

```diff
@@ -16,24 +16,23 @@ def create
       end
 
       StatsD.increment("#{stats_key}.success")
-      Rails.logger.info "ClaimID=#{claim.id} RPO=#{claim.region} Form=#{claim.form_type}"
-      clear_saved_form("22-#{claim.form_type}")
-      render(json: claim)
+      Rails.logger.info "ClaimID=#{claim.id} RPO=#{claim.education_benefits_claim.region} Form=#{form_type}"
+      clear_saved_form(claim.form_id)
+      render(json: claim.education_benefits_claim)
     end
 
     private
 
-    def education_benefits_claim_params
-      allowed_params = params.require(:education_benefits_claim).permit(:form)
-      form_type = params[:form_type]
-      allowed_params[:form_type] = form_type if form_type.present?
+    def form_type
+      params[:form_type] || '1990'
```

Comment:
Since this is passed off to `constantize` should we verify membership in a set of form types?